InternetHospital.BusinessLogic/Interfaces/IDoctorService.cs

```diff
@@ -5,7 +5,8 @@ namespace InternetHospital.BusinessLogic.Interfaces
 {
     public interface IDoctorService
     {
-        (IEnumerable<DoctorModel> doctors, int count) GetFilteredDoctors(DoctorSearchParameters queryParameters);
+
+      (IEnumerable<DoctorModel> doctors, int count) GetFilteredDoctors(DoctorSearchParameters queryParameters);
         ICollection<SpecializationModel> GetAvailableSpecialization();
```

Comment:
Where is your Get method from service?

Reply:
It seems like i have overfixed code last night.